src/lib.rs

```diff
@@ -112,18 +112,18 @@
 //!
 //! loop {
 //!     let keys = if pin.is_high().unwrap() {
-//!             &[Keyboard::A]
+//!             [Keyboard::A]
 //!         } else {
-//!             &[Keyboard::NoEventIndicated]
+//!             [Keyboard::NoEventIndicated]
```

Comment:
Please also make these change to `README.md`.

Is there a good way to keep these in sync?

Reply:
Done. Unfortunately I don't know of any good way to sync these.